pkg/image/api/helper.go

```diff
@@ -847,3 +848,41 @@ func SetContainerImageEntrypointAnnotation(annotations map[string]string, contai
 func LabelForStream(stream *ImageStream) string {
 	return fmt.Sprintf("%s/%s", stream.Namespace, stream.Name)
 }
+
+// ImageHasSignature returns true if the signature identified by type and blob is already stored in the image.
+func ImageHasSignature(image *Image, sType string, blob []byte) bool {
```

Comment:
Move these to the next pull? I have questions about them, like: "why don't I use a signature type", that probably don't need to be addressed here.